hq/app/services/MetricService.scala

```diff
@@ -74,7 +78,6 @@ class MetricService(
       else Duration.Zero
 
     val cloudwatchSubscription = Observable.interval(initialDelay, 6.hours).subscribe { _ =>
-      logger.info("Posting new metrics to cloudwatch")
```

Comment:
Logging this here isn't that helpful, because we get this log line even if we skip due to problems with the data